views/help/support.pug

```diff
@@ -0,0 +1,9 @@
+extends ../layouts/help-layout
+
+block content
+
+  Title= Help
```

Comment:
Is this `Title` intentional, or is it supposed to be a div with a `.title` class?